Add Ceres products to SORA docs (#225)
sora-xor/sora-docs: src/ceres/token-locker.md

---
title: "Token Locker | SORA Docs"
head:
  - - meta
    - name: description
      content: "Ceres Token Locker is a decentralized application that gives project teams ability to lock their tokens."
  - - meta
    - name: keywords
      content: "Ceres, DeFi, token locker, Polkaswap, SORA network"
---

# Ceres Token Locker

> [Ceres Token Locker](https://dapps.cerestoken.io/token_locker) is a decentralized application that gives project teams ability 

Reviewer: ```suggestion
The [Ceres Token Locker](https://dapps.cerestoken.io/token_locker) is a decentralized application that gives project teams the ability 
```

> to lock their team/marketing/other tokens for some period in order to ensure investors that those tokens will not be sold.

Reviewer: ```suggestion
to lock their team/marketing/other tokens for pre-determined periods to ensure investors that those tokens will not be sold.
```
